ribbon/src/test/java/feign/ribbon/RibbonClientTest.java

```diff
@@ -123,6 +125,26 @@ public void urlEncodeQueryStringParameters() throws IOException, InterruptedExce
     assertEquals(recordedRequestLine, expectedRequestLine);
   }
 
+
+  @Test
+  // Issue-183
```

Comment:
nit I don't think this comment is required. the commit log already says this, and the need for this feature is obvious.